language-design.md

```diff
@@ -70,7 +231,33 @@ what ways do the themes apply to the study and creation of DSLs?
 
 **Response**
 
-
+As the title of the article, _C is Manly and Python is For n00bs_
+[Yang and Rabkin 2015] suggests, higher-level and less complex languages may
+be looked down upon culturally. This bias most likely applies to many DSLs
+as well because DSLs often have more limited, specialized syntax. However, 
+that limitation of  a DSL may not be the only reason it would be looked down 
+upon based on Yang and Rabkin. One of the points made in Yang and Rabkin is 
+that “programmers confuse their strong views about languages with their views 
+about users of the languages” [Yang and Rabkin 2015]. This suggests another 
+reason why DSLs might be looked down upon by the larger programming community. 
+Because DSLs have a specific domain, they might have users who are not 
+stereotypical programmers and therefore looked down upon. There is even an 
+example of DSL that is looked down on for this exact reason in Yang and Rabkin, 
+“Software engineers sometimes deride statistical analysis languages like R or 
+SAS as ‘not real programming’ ” [Yang and Rabkin 2015]. Like other DSLs, R is 
+not typically used by software engineers and that likely contributes to the 
+bias against it. As described in the article, “Languages often spread in 
+disjoint real-world communities, making it easy for false perceptions to 
+arise about a language’s user base” [Yang and Rabkin 2015]. This probably 
+applies to DSLs even more so than for other languages since DSLs are often 
+targeted at a specific group of users. 
+
+Another aspect of DSLs that this article may touch on is their creation. 
+Creating a DSL might be regarded as “manly” since it is challenging and 
+might be done in a language like C, which is “Manly” [Yang and Rabkin 2015].
+A DSL also implies that someone took the trouble to make things (probably) 
+easier for the user of the language, which is in line with the language
+being looked down upon in the same way Python might be. 
```

Comment:
I agree, this is a great point! It makes the job of programmers extremely confusing if doing the job properly is looked down upon/less valuable. 😕